starter-code/iteration-1/main.ts

```diff
@@ -1,11 +1,28 @@
 // Iteration 1
 // 1. Create an array of strings
+const tasks = [String]
 
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
-
+const addTask = (newTask) => {
```

Comment:
```const addTask = (newTask): number => {```

Reply:
Remember that you can add types to the function. This one should only be able to return a number.